R/generateSyntheticData.R

```diff
@@ -66,9 +94,90 @@ generateSyntheticData <- function(dataset, n.vars, samples.per.cond, n.diffexp,
   
   ## Generate a unique ID for the data set
   uID <- paste(sample(c(0:9, letters, LETTERS), 10, replace = TRUE), collapse = "")
+  
+  ## Checks for phylogenetic tree
+  use_tree <- !is.null(tree) # If tree is specified, use it.
```

Comment:
Extra check for format of optional arguments.

Define `use_tree` and `use_length` variables that are used to alter the behavior of the function. If both are FALSE, then the original behavior should be preserved.